### ISSUE TITLE
[OP] Add Min-Max scaling under Normalize() Class in ops.py

### ISSUE THREAD

**Issue by benfred** [NVIDIA/NVTabular#34]
<a href="https://github.com/rnyak"><img src="https://avatars1.githubusercontent.com/u/16246900?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [rnyak](https://github.com/rnyak)**
_Tuesday May 19, 2020 at 16:27 GMT_
_Originally opened as https://github.com/rapidsai/recsys/issues/145_

----

**Is your operator request related to a problem? Please describe.**
Min-Max scaling is one of the most common scaling techniques that is used for data pre-processing.

**Describe the solution you'd like**
A clear and concise description of the operation you'd like to perform on the column.  Please include:
 - Type: Preprocessing
 - input column type(s): Continuous numeric 
 - output column type(s): Float numeric
 - Expected transformation of the data after application: Data is transformed to be within [0-1] range.

**Optional: Describe operation stages in detail***
Statistics per chunk: Ex. compute the min and max of the column
Statistics combine: NA
Apply: (value-min)/(max-min)

**Additional context**

Example code for Sklearn `MinMaxScaler` class can be found here:
https://github.com/scikit-learn/scikit-learn/blob/fd237278e/sklearn/preprocessing/_data.py#L200

**Comment by gabrielspmoreira** [comment]
For scaling ops, I believe we should store the original statistics in a metadata file and output them after the pipeline, i.e. storing the original min and max values for Min-Max and the original mean and std for ZNorm.
This is required for inference (we should apply the same scaling for prediction) and also to recover features in the original scale (e.g. converting the output of a regression model to a prediction in the same scale as the label)

**Comment by benfred** [comment]
@gabrielspmoreira We have the ability to load/save stats (including min/max ) already https://github.com/NVIDIA/NVTabular/blob/master/nvtabular/workflow.py#L862 . Its definitely required =).

@rynak for min-max scaling, is the correct calculation ```(value-mean)/(max-min)``` ? That won't transform values to a [0-1] range, I think what we should be looking for is ```(value-min)/(max-min)``` instead (substracting the min rather than mean)

**Comment by rnyak** [comment]
@benfred yes correct, fixing the typo there.  it should be :

`(value-min)/(max-min)`
